### PR TITLE
Updates for Seqtk

### PR DESCRIPTION
**Pull request by DLBPointon**
Updated the 6 seqtk modules. All are now 1.4 rather than a mix of 1.3 and 1.4.

Added a tuple in and out to subseq.

Added nf-tests.

4 modules had no stub, now added and tested.

### CODE REVIEW

File: modules/nf-core/seqtk/cutn/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        }
+
+        then {
+            assertAll (
+                { assert process.success },
+                { assert snapshot(process.out).match() },
+                { assert snapshot(
+                            file(process.out.bed[0][1]).name
+                                ).match("genome_cut")
+                }
```
Comment (maxulysse) [inline modules/nf-core/seqtk/cutn/tests/main.nf.test:32]: I don't think you need that if you already snapshot `process.out`

Comment (DLBPointon) [reply modules/nf-core/seqtk/cutn/tests/main.nf.test:32]: I thought that the above was specifically for the file where as the `process.out` was there to catch everything. Happy to be wrong and remove the `process.out` though.

Comment (DLBPointon) [reply modules/nf-core/seqtk/cutn/tests/main.nf.test:32]: Hi @maxulysse,
You were right, so i've updated to just use process out as that is better for static output. The other method is for when output has content varying between runs. Just a misunderstanding on my part to include both.

There's now an issue with the runner but that's under investigation.

**Comment by DLBPointon** [comment]
@maxulysse Could I have an approval if the above is ok?

**Comment by DLBPointon** [comment]
NF-Test API limit hit once again.

**Pull request merged**